dynawo/CMakeLists.txt

```diff
@@ -420,10 +417,7 @@ write_basic_package_version_file(
   COMPATIBILITY AnyNewerVersion
 )
 
-install(FILES
-    ${CMAKE_CURRENT_BINARY_DIR}/dynawo-config-version.cmake
-  DESTINATION ${SHAREDIR_NAME}
-)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/dynawo-config-version.cmake DESTINATION ${SHAREDIR_NAME})
```

Comment:
Can we remove those unrelated modifications?

Reply:
ok